src/screens/profile-screen/RepostsTab.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from 'react'
+
+import { feedActions } from 'audius-client/src/common/store/pages/profile/lineups/feed/actions'
+import { getProfileFeedLineup } from 'audius-client/src/common/store/pages/profile/selectors'
+import { Text } from 'react-native'
+
+import { Lineup } from 'app/components/lineup'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+
+import { EmptyTab } from './EmptyTab'
+import { getProfile } from './selectors'
+
+export const RepostsTab = () => {
+  const { profile } = useSelectorWeb(getProfile)
+  const lineup = useSelectorWeb(getProfileFeedLineup)
+  const dispatchWeb = useDispatchWeb()
+
+  const handlePlayTrack = useCallback(
+    (uid: string) => {
+      dispatchWeb(feedActions.play(uid))
+    },
+    [dispatchWeb]
+  )
+
+  const handlePauseTrack = useCallback(() => {
+    dispatchWeb(feedActions.pause())
+  }, [dispatchWeb])
+
+  if (!profile) return null
+
+  if (profile.repost_count === 0) {
+    return (
+      <EmptyTab>
+        <Text>{"You haven't reposted anything yet"}</Text>
+      </EmptyTab>
+    )
+  }
+
+  return (
+    <Lineup
+      actions={feedActions}
+      lineup={lineup as any}
```

Comment:
going to work on the typing here, current issue is
```
Property '_loading' is missing in type 'LineupStateTrack<{ id: number; }>' but required in type 'LineupItem'.ts(2322)
[types.ts(14, 3): ]()'_loading' is declared here.
```
any immediate thoughts on what to do here?

Reply:
I think we should remove `_loading` from `LineupItem` and do 

```ts
    if ((item as LoadingLineupItem)._loading) {
```

here: https://github.com/AudiusProject/audius-mobile-client/blob/master/src/components/lineup/Lineup.tsx#L194

Reply:
mind double checking the route i took?

Reply:
I think that's fine, but `_isLoading` will never exist on a lineup item. I added that field as a flag for use inside the Lineup logic to display extra skeletons

But I can go back and clean that up when tackling the rest of the Lineup changes

Reply:
ohh gotcha